js/components/AlertsComponent.js

```diff
@@ -98,13 +100,13 @@ export default class AlertsComponent extends React.Component {
                 )
               }}
             >
-              <Text>Disabilita Inbox</Text>
+              <Text>{I18n.t('inbox.enableButton')}</Text>
             </Button>
             <MessageBoxComponent />
           </View>
         ) : (
           <View>
-            <Text>Abilita l'inbox per visualizzare i messaggi</Text>
+            <Text>{I18n.t('inbox.line1')}</Text>
```

Comment:
`line1` is not very descriptive as an identifier, think about the guy that is going to do the translation in other languages... perhaps `inbox.enableCallToActionDescription`

Reply:
fixato